compiler/qsc_rir/src/passes/build_dominator_graph/tests.rs

```diff
@@ -86,13 +74,38 @@ fn test_dominator_graph_sequential_blocks_dominated_by_predecessor() {
 #[test]
 fn test_dominator_graph_branching_blocks_dominated_by_common_predecessor() {
```

Comment:
Test names don't really need to have test in the name, they are already attributed with `#[test]`.